### ISSUE TITLE
Is this project dead ?

### ISSUE THREAD

**Issue by sylr** [AGWA/git-crypt#131]
Hi,

I am currently investigating solutions to crypt certificates, private keys ... etc in git repo and came across this project.

I was considering it until I noticed last commit in master occurred more than 1 year ago and that PR stacking up does not seem to be considered.

@AGWA Can we have a status update about this project ?

Cheers.

**Comment by alerque** [comment]
Even as a long time and current user of this system I'm starting to get concerned about the future of the project. Obviously the lack of updates or feedback from the developer recently is the primary issue here, however the continuing backlog of PR's and issues suggests that the community is still interested.

@AGWA Would you be open to allowing some help into this project? I have some more upcoming projects that are going to need this and am considering whether to spend some time learning and implementing a different solution or whether I could invest some time and effort into helping this one along. If you are interested in allowing somebody like me to help with issue triage and handling some of the more basic community contributions that might take some of the load off so that any time you do have can be spent on more hard core development tasks.

I'm well aware that anybody including myself could choose to fork and run with this, but my preference for OS projects is to take smaller steps and foster contributions to the original project rather than forging off in some other direction ­— but cases like this it's going to take either you having time to manage community contributions or letting somebody else step in and help pull things together.

**Comment by amorriscode** [comment]
We just got a new release on the 26th of November so it looks okay to me.

**Comment by happycollision** [comment]
How about now? :wink: 

Fewer updates as time goes on could be more of an indication that a codebase is reaching "peak robustness" I suppose. So that is not necessarily a bad thing. Just hard to tell from the outside.

**Comment by alerque** [comment]
@happycollision In spite of the concern I had (and sort of still have), I'm still using it in production and it works fine. You have a point about robustness. In this case I do think there are more rough edges that could be filed off, but at some point when a tool only does one basic function and does it properly, it doesn't merit constant tinkering.

**Comment by sylr** [comment]
@alerque Agreed but it's not a reassuring for users that the maintainer is awol

**Comment by alerque** [comment]
@sylr I don't think it's fair to say the maintainer is AWOL. They've at least been [committing here on Github as recently as a couple weeks ago](https://github.com/SSLMate/caa_helper/commit/4e75cde5f7770bbda07adac10a03ebb2041b8cff) and there are no known critical security issues or blocking bugs that haven't been addressed.

**Comment by esc** [comment]
yes/no/maybe?

**Comment by esc** [comment]
@AGWA are you still around?

**Comment by AGWA** [comment]
Yes, I'm still around.

**Comment by happycollision** [comment]
I am so sorry that I inadvertently revived this issue. 🤦‍♂️

**Comment by alerque** [comment]
@AGWA I think we can probably close this issue. Two years and a release later, plus confirmation that nobody is dead, pretty much sets the stage. I don't think there is anything left to answer. Nobody can predict what turn of events life might play on your or others, but it seems like the status quo is pretty established for the time being — a slow trickle of maintenance releases. Fair enough?

**Comment by sylr** [comment]
@alerque I beg to differ, @AGWA showed that he is still around but did not said anything about the state of this project.

Last commit is more than one year ago and lot of PR are still unanswered.

I am not blaming @AGWA if he does not want to maintain it anymore I'm totally fine with it but he needs to state it so that users know what they can expect.

**Comment by natanverdes** [comment]
Hello @AGWA . I think the community wants to solve a lot of problems that this project have. We should close or merge the Merge Requests. Please open the project to some well-known developers of this project. Thank you and regards

**Comment by alerque** [comment]
Time to ping on this. The number of PRs piling up that appear to solve a lot of problems but aren't getting reviewed (whether to be rejected, modified, or cleaned up and accepted) is concerning. If you're not willing to open this up at least a little to a few trusted maintainers it might be time to start a community fork. I understand your concerns about it getting out of hand, but as a very long time user the lack of progress fixing known issues is starting to seriously grate again. Thoughts?

**Comment by fasibio** [comment]
Two years later... same situation, same question ?

**Comment by alerque** [comment]
@sylr I would appreciate the actual repository owner being the one to set the status on this issue to closed if they actually don't plan on making any changes, your impressions of the situation shouldn't be the *de facto* status shown to others waiting for news on this topic. Thanks.
